CONTRIBUTING.md

```diff
@@ -65,82 +65,63 @@ Once review is given, Maintainers and SIG members may indicate merge readiness w
+> | [Niladri Halder](https://www.linkedin.com/in/niladrih/ "Senior Engineer") | :rocket: <kbd>**[@niladrih](https://github.com/niladrih "Niladrih Halder")**</kbd> | ![](https://github.com/openebs/community/blob/develop/images/flags/de_je/in.png "I am based in Bengaluru, Karnataka, India (GMT+5:30) Timezone") | ```Maintainer``` | PR's / Issues |
+> | [Ed Robinson](https://www.linkedin.com/in/edrob/ "CNCF Head Liason") | :dog: <kbd>**[@edrob999](https://github.com/edrob999 "Ed Robinson")**</kbd> | ![](https://github.com/openebs/community/blob/develop/images/flags/ni_tn/nz.png "I'm a Kiwi from New Zealand. I am based in San Francisco, USA (GMT-7) Timezone")  &nbsp; ![](https://github.com/openebs/community/blob/develop/images/flags/to_zw/us.png "I am based in San Francisco, USA (GMT-7) Timezone") | <kbd>**CNCF Primary Liason**</kbd> | CNCF / Biz |
+> | [Tiago Castro](https://www.linkedin.com/in/tiago-castro-3311453a/ "Chief Architect") | :zap: <kbd>**[@tiagolobocastro](https://github.com/tiagolobocastro "Tiago Castro")**</kbd> | ![](https://github.com/openebs/community/blob/develop/images/flags/ni_tn/pt.png "I'm Portugueses. I'm based in London, UK (GMT+1) Timezone") &nbsp; ![](https://github.com/openebs/community/blob/develop/images/flags/de_je/gb.png "I am based in London, UK (GMT+1) Timezone") | <kbd>**Admin**</kbd>, ```Maintainer``` |  PR's / Issues / Arch |
+> | [David Brace](https://www.linkedin.com/in/dbrace/ "Head of Product Mgmt & Strategy") | :star: <kbd>**[@orville-wright](https://github.com/orville-wright "Dave Brace")**</kbd> | ![](https://github.com/openebs/community/blob/develop/images/flags/ni_tn/nz.png "I'm a Kiwi from New Zealand. I am based in San Francisco, USA (GMT-7) Timezone") &nbsp; ![](https://github.com/openebs/community/blob/develop/images/flags/de_je/hu.png "I'm also Hungarian. I'm based in San Francisco, USA (GMT-7) Timezone") &nbsp; ![](https://github.com/openebs/community/blob/develop/images/flags/to_zw/us.png "Yes, I'm also an American. I'm based in San Francisco, USA (GMT-7) Timezone") | <kbd>**Admin**</kbd>, ```Maintainer``` | Prod / Issues / Biz |
 
-Our maintainers are:
+<BR>
 
-- [@gila][members-gila] - [@mayadata-io][maya-data]
-- [@jkryl][members-jkryl] - [@mayadata-io][maya-data]
-- [@GlennBullingham][members-glennbullingham] - [@mayadata-io][maya-data]
+> [!Important]
+> When creating an ISSUE or a PR. Please TAG one (or more) of the Maintainers.
 
 Our Special Interest Groups (SIGs) are:
+| Functional area | Name | @GitHub | Are of Expertese | Product specialization |
+| :--- | :--- | :--- | :--- | :--- |
+| ``` Entire Product ``` | Tiago Castro | @tiagolobocastro | Architect / Eng | Mayastor |
+| ``` Data Plane ``` | Dimitry Savitskiy | @dsavitskiy | Architect / Eng | Mayastor |
+| ``` Data Plane ``` | Jonathan Teh | @jonathan-teh | Eng | Mayastor |
```

Comment:
Jonathan Teh, Mikhail Tcymbaliuk and Paul Yoong are no longer maintainers AFAIK.